CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+# 0.2.0
```

Comment:
Yes as I need to update that in main during the actual release process,  to prevent someone from publishing the package with that version by accident